exporter/signalfxexporter/config.go

```diff
@@ -62,6 +63,14 @@ type Config struct {
 	LogDimensionUpdates bool `mapstructure:"log_dimension_updates"`
 
 	splunk.AccessTokenPassthroughConfig `mapstructure:",squash"`
+
+	// SendCompatibleMetrics specifies if metrics must be sent in a format backward-compatible with
+	// SignalFx naming conventions, "false" by default.
+	SendCompatibleMetrics bool `mapstructure:"send_compatible_metrics"`
+
+	// TranslationRules defines a set of rules how to translate metrics to a SignalFx compatible format
+	// If not provided explicitly, the rules defined in translations/config/default.yaml are used.
+	TranslationRules []translation.Rule `mapstructure:"translation_rules"`
```

Comment:
Could you also update the README to include these options?

Reply:
Updated